### DISCUSSION TITLE
dinitcheck: `libc++abi: terminating due to uncaught exception of type dinit_load::setting_exception`

### DISCUSSION THREAD

**Discussion by mobin-2008** [davmac314/dinit#135] [General]
Hi! I writed this service for runs by [turnstile](https://github.com/chimera-linux/turnstile) on every login/logout.
```
type = scripted
command = /bin/bash -c "echo Mobin has logged in at $(date)"
stop-command = /bin/bash -c "echo "Mobin has logged out at $(date)"
logfile = /home/mobin/.login_history
```
as you see, `stop-command` string is broken. It's contain a unclosed `"` in `echo` command. I expect `dinitcheck` show me a error about this but it's terminates by system via `SIGABRT`:
```
➜ dinitcheck login_history
Checking service: login_history...
libc++abi: terminating due to uncaught exception of type dinit_load::setting_exception
[1]    7458 abort      dinitcheck login_history
```
I posted it in here because I'm not sure it's a "bug".

Regards

**Comment by davmac314** [comment]
It's a bug, created as #136
